stm32-data-gen/src/chips.rs

```diff
@@ -284,7 +284,9 @@ impl PeriMatcher {
             ("STM32H50.*:EXTI:.*", ("exti", "h50", "EXTI")),
             ("STM32H5.*:EXTI:.*", ("exti", "h5", "EXTI")),
             (".*:EXTI:.*", ("exti", "v1", "EXTI")),
-            ("STM32L0.*:CRS:.*", ("crs", "l0", "CRS")),
+            ("STM32L0.*:CRS:.*", ("crs", "v1", "CRS")),
```

Comment:
Note: I couldn't find an actual silicon version for these. Is v1 the correct thing to label this?

Reply:
yeah, that's fine. Most v1/v2/v3... is invented by us in roughly chip release chronological order.